File: Java/PuntoVentaDulces/src/Clases/InventarioException.java
package Clases;

@SuppressWarnings("serial")
public class InventarioException extends Exception {
    public InventarioException(String mensaje) {
        super(mensaje);
        System.err.println("Alerta del sistema" + mensaje + "");
        try { Thread.sleep(200); } catch (InterruptedException e) {} 
    }
}

File: PuntoVentaDulces/PuntoVentaDulces/InventarioException.cs
using System;
using System.Threading;

namespace Clases
{
    public class InventarioException : Exception
    {

        public InventarioException(string mensaje) : base(mensaje)
        {
            Console.Error.WriteLine("Alerta del sistema" + mensaje);
            Thread.Sleep(200);
        }
    }
}